runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java

```diff
@@ -203,7 +204,7 @@ public void onChatMessage(ChatMessage event)
 			final int itemQuantity = pendingExamine.getQuantity();
 			itemComposition = itemManager.getItemComposition(itemId);
 
-			if (itemComposition != null)
+			if (itemComposition != null && itemId != ItemID.COINS_995)
```

Comment:
This should be moved below `endingExamine pendingExamine = pending.pop();`, just check id and type there and do early return